logrecorder-modules/logrecorder-logback/src/main/kotlin/io/github/logrecorder/logback/LogbackLogRecord.kt

```diff
@@ -47,7 +47,7 @@ internal class LogbackLogRecord : LogRecord {
             },
             message = value.formattedMessage,
             marker = value.marker?.toString(),
-            properties = value.mdcPropertyMap.toMap(),
+            properties = value.mdcPropertyMap.toMap() + value.keyValuePairs.orEmpty().associate { kvp -> kvp.key to kvp.value.toString() },
```

Comment:
Writing `kvp.value.toString()` may lead to a `NullPointerException`
To prevent that you could write `"${kvp.value}"`, that way `null` but be represented as the String `"null"`.